tools/support_matrix/lib/support_matrix/pdf_formatter.rb

```diff
@@ -0,0 +1,194 @@
+require "prawn"
+require "prawn/table"
+class PdfFormatter
+  attr_reader :document
+
+  def initialize
+    @document = Prawn::Document.new(:page_layout => :landscape)
+  end
+
+  def write(matrix, path)
+    full_path = File.join(path, "Machinery_support_matrix.pdf")
+
+    legend
+    integration_tests_sheet(matrix)
+    unit_tests_sheet(matrix)
+    runs_on_sheet(matrix)
+    sections
+
+    @document.render_file(full_path)
+    full_path
+  end
+
+  private
+
+  def sections
+    @document.outline.define do
+      section("Legend", destination: 1)
+      section("Integration Tests") do
+        page title: "Inspect", destination: 2
+        page title: "Build", destination: 3
+        page title: "Export", destination: 4
+        page title: "Analyze", destination: 5
+        page title: "Deploy", destination: 6
+      end
+      section("Unit Tests", destination: 7)
+      section("Runs On", destination: 8)
+    end
+  end
+  def legend
```

Comment:
Assignment Branch Condition size for legend is too high. [15.03/15]<br>Use empty lines between defs.